### PR TITLE
Value/Simplex Noise

### PR DESCRIPTION
**Pull request by adodge**
Node to generate noise textures.

image

### CODE REVIEW

**Comment by theflyingzamboni** [comment]
Quick note, we've recategorized all basic image-generating nodes into Image > Make Images.

File: backend/src/nodes/nodes/image/create_noise.py [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+                {
+                    "enum": 6,
+                    "conditions": [
+                        FractalMethod.PINK.value,
+                        FractalMethod.PINK.value,
+                        FractalMethod.PINK.value,
+                        FractalMethod.PINK.value,
+                    ],
+                },
+            )(
```
Comment (RunDevelopment) [inline backend/src/nodes/nodes/image/create_noise.py:72]: I'd recommend using the new `conditional_group` function.
```suggestion
            conditional_group(enum=6, condition=FractalMethod.PINK.value)(
```

File: backend/src/nodes/nodes/image/create_noise.py [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+                BoolInput("Increment Seed", default=True).with_id(13),
+            ),
+        ]
+        self.outputs = [
+            ImageOutput(
+                image_type=expression.Image(
+                    width="Input0",
+                    height="Input1",
+                    channels="1",
+                )
```
Comment (RunDevelopment) [inline backend/src/nodes/nodes/image/create_noise.py:89]: Using the channels parameter on `ImageOutput` is preferred.
```suggestion
                image_type=expression.Image(
                    width="Input0",
                    height="Input1",
                ),
                channels=1,
```

**Comment by RunDevelopment** [comment]
About the CI: I recently added isort to organize imports. If you are using VSCode, just install the extension and hit Ctrl+S on the offending file.

File: backend/src/nodes/nodes/image/create_noise.py (hunk 1 above)
Comment (adodge) [reply backend/src/nodes/nodes/image/create_noise.py]: Oh, cool!

**Review by joeyballentine — APPROVED** [review]

**Pull request merged**